scripts/jenkins/qa/cloud-mkphyscloud-qa-tests-trigger.yaml

```diff
@@ -107,3 +112,22 @@
                     scenario_name=$scenario_name
                     scenario_job_name=$scenario_job_name
                     scenario_build_number=$scenario_build_number
+
+      - conditional-step:
+          condition-kind: boolean-expression 
+          condition-expression: ${ENV,var="horizon-tests"}
+          steps:
+            - trigger-builds:
+                - project: cloud-mkphyscloud-qa-test-horizon-tests
+                  condition: ALWAYS
+                  block: true
+                  block-thresholds:
+                    build-step-failure-threshold: never
+                    failure-threshold: never
+                    unstable-threshold: never
```

Comment:
All the thresholds above should rather be set to `FAILURE` as once the job is triggered requiring horizon tests to be run (boolean  `horizon-tests` set to true), we want the build to be shown as failed if those horizon tests fail.